lib/components/chat_history/twitch/poll_indicator.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+
+class PollIndicatorWidget extends StatelessWidget {
+  final PollChoiceData data;
+  final bool isCompleted;
+  final int maxVotes;
+  const PollIndicatorWidget({
+    Key? key,
+    required this.data,
+    required this.isCompleted,
+    required this.maxVotes,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Instead of passing `percentage` through the model, calculate it here:

```
final percentage = data.votes / math.min(1, this.maxVotes);
```

Reply:
I changed it to ```final percentage = data.votes / math.max(1, this.totalVotes);```